test/HcalDigiPipelineTest.cxx

```diff
@@ -0,0 +1,259 @@
+
+#include "catch.hpp" //for TEST_CASE, REQUIRE, and other Catch2 macros
+
+#include "DetDescr/HcalID.h" //creating unique cell IDs
+#include "Framework/ConfigurePython.h"
+#include "Framework/EventProcessor.h"
+#include "Framework/Process.h"
+
+namespace hcal {
+namespace test {
+
+/**
+ * Energy deposited by one MIP on average
+ * [MeV]
+ */
+static const double MIP_ENERGY = 4.66;
+
+/**
+ * Conversion between voltage and deposited energy
+ * [MeV/mV]
+ */
+static const double MeV_per_mV = MIP_ENERGY/(5*68); // 0.013 MeV/mV
+
+/**
+ * Maximum percent error that a single hit
+ * can be reconstructed with before failing the test
+ * if above the tot threshold.
+ *
+ * Comparing energy deposited in Silicon that was
+ * "simulated" (input into digitizer) and the reconstructed
+ * energy deposited output by reconstructor.
+ */
+static const double MAX_ENERGY_PERCENT_ERROR_DAQ_TOT_MODE = 2.;
```

Comment:
If TOT has not been implemented yet, comment here that this will never be tested.